readme.org

```diff
@@ -152,8 +152,6 @@ See the =make-instance*= documentation for more examples and details.
 
 ** Changes from =defclass-star=
 
-- Renamed =defclass*= to =define-class=.
-- Renamed =defcondition*= to =define-condition*=.
```

Comment:
Still need to mention aliases, otherwise the user might wonder why we have multiple names.

Reply:
Yes. Done in c9b7b9d1fa6346caf0b9c3790713beb23a58bbd5!